test/e2e/utils/testenv/testenv.go

```diff
@@ -72,6 +73,13 @@ func GetPollInterval() string {
 	return pollInterval
 }
 
+func GetDeviceName() string {
+	if devName, ok := os.LookupEnv("E2E_DEVICE_NAME"); ok {
```

Comment:
IIUC, this should be `DEVICE_RESOURCE_NAME`. We are creating environment variable when deploying the device plugin [here](https://github.com/k8stopologyawareschedwg/resource-topology-exporter/pull/120/commits/91f7d2059d1c92acebfde4f1e3829c2618050eff#diff-e19ff91c1b014f024d0a56149f75ceb456dbb075243fe8971143611def27140bR20) where the key is  `DEVICE_RESOURCE_NAME`. Is `E2E_DEVICE_NAME` meant to be a separate environment variable that the we are supposed to create? Can you please clarify if I am missing something here?

Reply:
The ```DEVICE_RESOURCE_NAME``` is defined inside the device-plugin pod and we don't have an access to it in the test scope.
We need the ```E2E_DEVICE_NAME``` so we could tell the test the exact name to look for in case we would import the new test in other packages.
Like we are doing here:
https://github.com/openshift-kni/numaresources-operator/blob/main/test/e2e/rte/rte_e2e_suite_test.go#L32

Reply:
Ack, thanks for clarifying this.